### DISCUSSION TITLE
Export to csv file using special column delimiter

### DISCUSSION THREAD

**Discussion by Paul-Bukowski** [dbeaver/dbeaver#22885] [Q&A]
Is there a way to specify a hexadecimal column delimiter when I export from DBeaver to a csv file? I'm trying to use my favorite delimiter which is \0 (also known as \x00) or the NUL character.
Thanks !
Paul

**Comment by LonwoLonwo** [comment]
Hello @Paul-Bukowski 

Yes, you can specify a column delimiter in the data transfer settings. Like this.

![2024-02-21 11_38_10-Data Transfer](https://github.com/dbeaver/dbeaver/assets/45152336/6ba98ba1-c3f0-4108-bd2c-5c8ab32abb3f)

Or is it not working for you?

**Reply by Paul-Bukowski** [reply]
Hi @LonwoLonwo LonwoLonwo,
Thanks for the reply, but I'm aware of the comma as 1of the choices for the delimiter. I'm trying to use a hexadecimal non-printable character for the delimiter. Any idea how to use a custom delimiter like I described? The example I have in my post is the \0 NUL character, also known as CHR(0) or \x00

**Reply by LonwoLonwo** [reply]
You can use any custom symbol instead of a comma in this field.

**Reply by Paul-Bukowski** [reply]
How would I write that custom symbol in that field (in hexadecimal)?
\0
\x00

I've tried many things and it doesn't seem to work.
Thanks,
Paul

**Reply by LonwoLonwo** [reply]
Okay, that means that the hexadecimal value in this field is not supported.

**Reply by Paul-Bukowski** [reply]
Ok, thank you for your time !
